package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1729,7 +1729,99 @@ def radius_of_gyration(group, wrap=False, **kwargs):
     @warn_if_not_unique
     @_pbc_to_wrap
     @check_atomgroup_not_empty
-    def shape_parameter(group, wrap=False):
+    def gyration_moments(group, wrap=False, unwrap=None, compound='group'):
```

Comment:
Why is the default for `unwrap=None` if it's a boolean? Why not `False`?

Reply:
@richardjgowers @hmacdope do you know why we set `unwrap=None` as default instead of `unwrap=False`?